### PR TITLE
DKP-583 Add release notes for Docker Desktop 4.17.0

### PR DESCRIPTION
**Pull request by chaizeg**
### Proposed changes
Added Release Notes for Docker Desktop 4.17

### CODE REVIEW

File: desktop/release-notes.md [hunk 1]
```diff
@@ -24,34 +24,108 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Implemented the dangling filter on images for the containerd integration.
+- Fixed `docker ps` failing with containers whose images are no longer present with the containerd integration.
+
+#### For Mac
+
+- Fixed download of Registry Access Management policy on systems where the privileged helper tool `com.docker.vmnetd` is not installed.
+- Fixed a bug where `com.docker.vmnetd` could not be installed if `/Library/PrivilegedHelperTools` does not exist.
+- Fixed a bug where the "system" proxy would not handle "autoproxy" / "pac file" configurations.
+- Fixed a bug where vmnetd installation fails reading `Info.Plist` on case-sensitive file systems. Actual filename is `Info.plist`. Fixes [docker/for-mac#6677](https://github.com/docker/for-mac/issues/6677).
+- Fixed the start of the application on login not working.
```
Comment (aiordache) [inline desktop/release-notes.md:80]: We need also a release note for a bug where user is always prompted on startup to create the docker socket symlink.
Maybe we could have something like this:
```
Fix a bug where user is prompted to create the docker socket symlink on every startup.
Fixes [for-mac#6634](https://github.com/docker/for-mac/issues/6634)
```

File: desktop/release-notes.md [hunk 2]
```diff
@@ -24,34 +24,109 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+
+{% include release-date.html date="2022-02-27" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Image analysis results - you can now filter on vulnerability information.
```
Comment (aevesdocker) [inline desktop/release-notes.md:37]: I think these first four bullet points all relate to Docker Scout. Think we should reduce it to just `Docker Desktop now ships with Docker Scout. View and pull images from Artifactory repositories, get base image updates and recommended tags and digests, and filter your images on vulnerability information. To learn more, see [Docker Scout](/scout/index.md)`

**Review by aevesdocker — COMMENTED** [review]
Looks great @chaizeg, just a few nits

File: desktop/release-notes.md [hunk 3]
```diff
@@ -24,34 +24,109 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Fixed `docker ps` failing with containers whose images are no longer present with the containerd integration.
+
+#### For Mac
+
+- Fixed download of Registry Access Management policy on systems where the privileged helper tool `com.docker.vmnetd` is not installed.
+- Fixed a bug where `com.docker.vmnetd` could not be installed if `/Library/PrivilegedHelperTools` does not exist.
+- Fixed a bug where the "system" proxy would not handle "autoproxy" / "pac file" configurations.
+- Fixed a bug where vmnetd installation fails reading `Info.Plist` on case-sensitive file systems. Actual filename is `Info.plist`. Fixes [docker/for-mac#6677](https://github.com/docker/for-mac/issues/6677).
+- Fixed a bug where user is prompted to create the docker socket symlink on every startup. Fixes [docker/for-mac#6634](https://github.com/docker/for-mac/issues/6634).
+- Fixed the start of the application on login not working.
```
[the remark this answers is not in the document]
Comment (chaizeg) [reply desktop/release-notes.md:81]: The "Start Docker Desktop when you log in" setting in `General` was not working on Mac, It's a regression introduced in `4.16`.

[the remark this answers is not in the document]
Comment (aevesdocker) [reply desktop/release-notes.md:81]: Ok great, thanks. In that case, I'd suggest we say `Fixed a bug that caused the **Start Docker Desktop when you log in** setting to not work.`

File: desktop/release-notes.md (hunk 2 above)
Comment (aevesdocker) [reply desktop/release-notes.md:37]: @dvdksn am I right?

Comment (jonnyrobots) [reply desktop/release-notes.md:37]: Collapsing the bullets is fine, but I would just add Hub here for clarity:

> Docker Desktop now ships with Docker Scout. Pull and view analysis for images from **Docker Hub and** Artifactory repositories, get base image updates and recommended tags and digests, and filter your images on vulnerability information. To learn more, see [Docker Scout](/scout/index.md)

Comment (chaizeg) [reply desktop/release-notes.md]: @aevesdocker I'm not sure we have docs for `Docker Scout` yet 🤔 Is there another PR with `/scout/index.md` ?

Comment (aevesdocker) [reply desktop/release-notes.md]: yes :) https://github.com/docker/docs/pull/16679/files

Comment (aevesdocker) [reply desktop/release-notes.md]: The build will fix itself when the Scout PR is merged.

**Comment by dvdksn** [comment]
if you rebase now, the htmlproofer validation should become green.

File: desktop/release-notes.md [hunk 4]
```diff
@@ -24,34 +24,106 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+
+{% include release-date.html date="2022-02-27" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Docker Desktop now ships with Docker Scout. Pull and view analysis for images from Docker Hub and Artifactory repositories, get base image updates and recommended tags and digests, and filter your images on vulnerability information. To learn more, see [Docker Scout](/scout/index.md).
```
Comment (aevesdocker) [inline desktop/release-notes.md:37]:
```suggestion
- Docker Desktop now ships with Docker Scout. Pull and view analysis for images from Docker Hub and Artifactory repositories, get base image updates and recommended tags and digests, and filter your images on vulnerability information.
```

File: desktop/release-notes.md [hunk 5]
```diff
@@ -24,34 +24,106 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+
+{% include release-date.html date="2022-02-27" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Docker Desktop now ships with Docker Scout. Pull and view analysis for images from Docker Hub and Artifactory repositories, get base image updates and recommended tags and digests, and filter your images on vulnerability information.
```
Comment (aevesdocker) [inline desktop/release-notes.md:37]:
```suggestion
- Docker Desktop now ships with Docker Scout. Pull and view analysis for images from Docker Hub and Artifactory repositories, get base image updates and recommended tags and digests, and filter your images on vulnerability information. To learn more, see [Docker Scout](/scout/index.md)
```

File: desktop/release-notes.md (hunk 4 above)
Comment (aevesdocker) [inline desktop/release-notes.md:37]:
```suggestion
- Docker Desktop now ships with Docker Scout. Pull and view analysis for images from Docker Hub and Artifactory repositories, get base image updates and recommended tags and digests, and filter your images on vulnerability information. 
```

**Review by aevesdocker — APPROVED** [review]

**Pull request merged**